src/node_contextify.h

```diff
@@ -54,12 +54,19 @@ class ContextifyContext {
   static ContextifyContext* Get(const v8::PropertyCallbackInfo<T>& args);
 
  private:
+  struct CompileFnEntry {
```

Comment:
I'm sorry, but I don't understand what the need for this and `WeakCallbackCompileFn` is, since neither the scripts nor the modules are handled similarly. The way I originally implemented dynamic modules, it worked without the need for these.